src/app/jobs/weatherevents.job.ts

```diff
@@ -17,18 +17,16 @@ interface IProperties {
 }
 
 export class WeatherEventsJob extends Job {
-  // Run once an hour
-  public interval: number = 1000 * 60 * 60;
-
-  public name: string = 'weather_events';
+  public name: string = 'Weather Events';
+  public interval: number = moment.duration(1, 'hours').asMilliseconds();
 
   constructor() {
     super();
   }
 
   public async execute(container: IContainer) {
-    const channel = (container.guildService.getChannel("weather_events")) as TextChannel;
-    const resp = await container.httpService.get("https://api.weather.gov/alerts/active?area=FL");
+    const channel = (container.guildService.getChannel('weather_events')) as TextChannel;
+    const resp = await container.httpService.get('https://api.weather.gov/alerts/active?area=FL');

```

Comment:
Store this link as a class constant
`private _ENDPOINT: string = '...'`

Reply:
Got it!